ui/command_buffer_model.cpp

```diff
@@ -90,7 +90,7 @@ QVariant CommandBufferModel::data(const QModelIndex &index, int role) const
             return int(Qt::AlignLeft | Qt::AlignVCenter);
     }
 
-    uint64_t node_index = (uint64_t)(index.internalPointer());
+    uint64_t node_index = index.internalId();
```

Comment:
For further refactoring, probably have
```cpp
inline uint64_t ToNodeIndex(const QModelIndex &index) { return index.internalId(); }
```
defined somewhere, then
```cpp
auto node_index = ToNodeIndex(index);
```
so eventually we can change `node_index` to a opaque type instead of `uint64_t` which could be implicitly casted.